src/lib/components/transaction/Transaction.svelte

```diff
@@ -61,10 +61,10 @@
         isValidXdr = xdr.TransactionEnvelope.validateXDR(transactionMessage.xdr, 'base64');
         tx = new Transaction(transactionMessage.xdr, CURRENT_NETWORK_PASSPHRASE);
 
-        if (tx.networkPassphrase === 'Test SDF Network ; September 2015') {
-            network = 'Testnet';
+        if (tx.networkPassphrase === CURRENT_NETWORK_PASSPHRASE) {
+            network = CURRENT_STELLAR_NETWORK;
         } else {
-            network = 'Public';
+            network = CURRENT_STELLAR_NETWORK;
         }
```

Comment:
this if condition has the same outcome, i.e. the if condition is not needed, you can simply do

network = CURRENT_STELLAR_NETWORK